src/main/webapp/app/stores/tabs.store.ts

```diff
@@ -0,0 +1,16 @@
+import { action, makeObservable, observable } from 'mobx';
+
+export class TabsStore {
```

Comment:
The same to this store. The stores folder should only include the ones that intended to be shared across different pages. If it's a store that manages the state for a specific component, it needs to be placed together with component.